Applications/Spire/Source/Ui/CalendarDatePicker.cpp

```diff
@@ -164,15 +250,30 @@ class CalendarDayLabel : public QWidget {
         set(border_color(QColor::fromRgb(0, 0, 0, 0))).
         set(TextColor(QColor::fromRgb(0xC8C8C8)));
       set_style(*this, std::move(style));
+      auto layout = new QHBoxLayout(this);
+      layout->setContentsMargins({});
+      layout->addWidget(m_label);
+      on_current(std::any_cast<date>(m_model->get_current()));
+    }
+
+  private:
+    std::shared_ptr<ListValueModel> m_model;
+    scoped_connection m_current_connection;
+    std::shared_ptr<DateModel> m_month_model;
+    TextBox* m_label;
+
+    void on_current(date day) {
+      m_label->get_model()->set_current(QString("%12").arg(day.day()));
```

Comment:
Why is the place marker "%12"?

Reply:
It looks like QString will match any placeholder up to the number of arg(...) calls, so even with this typo it still functions correctly.

Reply:
Yeah, it doesn't affect the function, but I think it would be better to assign the place marker starting from 1.